ios/Classes/ContextMenuOptions.swift

```diff
@@ -10,6 +10,8 @@ import Foundation
 class ContextMenuOptions: Options<NSObject> {
     
     var hideDefaultSystemContextMenuItems = false;
+    // var filterDefaultSystemContextMenuItems = [String]();
+    var filterDefaultSystemContextMenuItems = nil;
```

Comment:
```suggestion
    var filterDefaultSystemContextMenuItems : [String]? = nil;
```

Reply:
I am pretty new to Swift and cannot really understand your correction, but i suppose it is valid, so I let Lorenzo decide, I hope that is ok.

Reply:
I've cherry-picked all recent PRs to my fork repo, at lease the correction did compile without error. You can try if it works as you expected.

Reply:
The error is about that you can't make variables as nil without type defined.

Reply:
ok, thanks.